canonical-web-and-design/microk8s.io
# Build /compare page

## Done

Build `/compare` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check page against [copy doc](https://docs.google.com/document/d/1wWos3iWMTCqDbxZBDuSVqvY6ZwYux-goCrX0L0E5Weo/edit)


## Issue / Card

Fixes [#4649](https://github.com/canonical-web-and-design/web-squad/issues/4649)

## Screenshots
<img width="1440" alt="MicroK8s vs k3s vs Minikube ()" src="https://user-images.githubusercontent.com/57550290/151164783-451c8fce-c568-4175-a193-15b879ebeacf.png">
